docs/data/stashcache/install-origin.md

```diff
@@ -100,11 +102,11 @@ The origin service consists of the following systemd units:
 
 | **Software** | **Service name** | **Notes** |
 |--------------|------------------|-----------|
-| XRootD | `xrootd@stashcache-origin-server.service` | The xrootd daemon, which performs the data transfers |
-| XRootD | `cmsd@stashcache-origin-server.service` | The "cluster management service" daemon, which integrates the origin into the data federation.  |
-| Fetch CRL         | `fetch-crl-boot` and `fetch-crl-cron` | See [CA documentation](/common/ca#managing-fetch-crl-services) for more info |
```

Comment:
We don't need fetch-crl anymore?

Reply:
Admins don't need to start it as systemd handles the dependency.